airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -63,6 +63,7 @@
             "properties": {
               "method": {
                 "type": "string",
+                "const": "Standard",
                 "enum": ["Standard"],
```

Comment:
you can also remove the `enum` and `default` fields as a result of this new convention